santorini-common/src/command.rs

```diff
@@ -1,13 +1,11 @@
 use crate::{
     error::SantoriniError,
-    objects::{
-        state::State,
-        tower::{Dome, Level},
-    },
-    position::Position,
+    objects::{state::State, tower::*},
+    position::*,
 };
 
 pub trait Command {
```

Comment:
As discussed offline, let's add add a method to this trait with a default implementation that invokes `can_execute` and then `execute`.